File: ccc15j1.py
month,day=int(input()),int(input())
def check(m,d):
    if month == 2 and day == 18:return("Special")
    elif month < 2:return("Before")
    elif month == 2 and day < 18:return("Before")
    elif month > 2:return("After")
    elif month == 2 and day > 18:return("After")
print(check(m=month,d=day))

File: ccc15j1.cpp
#include <iostream>
using namespace std;
int main(){
    int month, day;
    cin >> month;
    cin >> day;
    if (month == 2 && day == 18){
        cout << "Special" << endl;
        return 0;
    } else if (month < 2){
        cout << "Before" << endl;
        return 0;
    } else if (month == 2 && day < 18){
        cout << "Before" << endl;
        return 0;
    } else if (month > 2){
        cout << "After" << endl;
        return 0;
    } else if (month == 2 && day > 18){
        cout << "After" << endl;
        return 0;
    }
    return 0;
};